### PR TITLE
feat: blog: Vue Uncategorized rules to have a better life

### PR DESCRIPTION
**Pull request by BobToninho**
In this blog post I explain a bunch of Vue related rules, their advantages and how I use them to improve my overall DX. I'm aware there's not a lot of copy, so let me know if something more is needed.

### CODE REVIEW

File: src/content/drafts/vue-rules-better-life.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+I like to keep the rule this way:
+
+```json
+{
+  "vue/component-name-in-template-casing": ["error", "PascalCase"]
+}
+```
+
+The PascalCase convention for component names helps distinguish more clearly a component from an actual HTML element. Additionaly, depending on the theme, your editor or IDE will highlight a component in PascalCase with a different color with respect to an actual HTML element. In my experience, I find this small tweak especially useful when managing big components.
```
Comment (amareshsm) [inline src/content/drafts/vue-rules-better-life.md:37]:
```suggestion
The PascalCase convention for component names helps distinguish more clearly a component from an actual HTML element. Additionally, depending on the theme, your editor or IDE will highlight a component in PascalCase with a different color with respect to an actual HTML element. In my experience, I find this small tweak especially useful when managing big components.
```

File: src/content/drafts/vue-rules-better-life.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+The [esling-plugin-vue](https://eslint.vuejs.org/) plugin divides its rules in the following categories:
+
+- Base
+- Essential
+- Strongly recommended
+- Recommended
+- Uncategorized
+- Extensions
+
+I spent some time digging into the *uncategorized* category to see what can be useful to help avoiding distractions in commits and, of course, to enforce the same style in all parts of a Vue project. In my opinion, all Vue projects should use the Recommended category, which enforces a lot (if not all) of the topics mentioned in the [style guide](https://vuejs.org/style-guide/).
```
Comment (amareshsm) [inline src/content/drafts/vue-rules-better-life.md:25]:
```suggestion
I spent some time digging into the *uncategorized* category to see what can be useful to help avoid distractions in commits and, of course, to enforce the same style in all parts of a Vue project. In my opinion, all Vue projects should use the Recommended category, which enforces a lot (if not all) of the topics mentioned in the [style guide](https://vuejs.org/style-guide/).
```

File: src/_includes/partials/author_bios/BobToninho.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Young freelance developer
```
Comment (nzakas) [inline src/_includes/partials/author_bios/BobToninho.md:1]: Please expand your bio here. We generally like the form “Bob Toninho is a…. He…. And I’m his spare time he….”

File: src/content/drafts/vue-rules-better-life.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Vue Uncategorized rules to have a better life
```
Comment (nzakas) [inline src/content/drafts/vue-rules-better-life.md:3]: Maybe a bit clearer?
```suggestion
title: Exploring useful uncategorized rules in the Vue ESLint plugin
```

File: src/content/drafts/vue-rules-better-life.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+teaser: "Some useful Uncategorized Vue rules that will make your daily life a little easier."
```
Comment (nzakas) [inline src/content/drafts/vue-rules-better-life.md:4]:
```suggestion
teaser: "The Vue ESLint plugin has several rules that are uncategorized and it turns out many will make your daily life a little easier."
```

File: src/content/drafts/vue-rules-better-life.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+authors:
+  - BobToninho
+categories:
+  - Case Studies
+tags:
+  - Guest Post
+  - Rules
+---
+
+## Introduction
```
Comment (nzakas) [inline src/content/drafts/vue-rules-better-life.md:14]: We don’t need a heading here. The first few paragraphs always considered the introduction. 
```suggestion
```

File: src/content/drafts/vue-rules-better-life.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+The [esling-plugin-vue](https://eslint.vuejs.org/) plugin divides its rules in the following categories:
```
Comment (nzakas) [inline src/content/drafts/vue-rules-better-life.md:16]:
```suggestion
The official Vue ESLint plugin, [eslint-plugin-vue](https://eslint.vuejs.org/), groups its rules into the following categories:
```

File: src/content/drafts/vue-rules-better-life.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+- Base
+- Essential
+- Strongly recommended
+- Recommended
+- Uncategorized
+- Extensions
+
+I spent some time digging into the *uncategorized* category to see what can be useful to help avoid distractions in commits and, of course, to enforce the same style in all parts of a Vue project. In my opinion, all Vue projects should use the Recommended category, which enforces a lot (if not all) of the topics mentioned in the [style guide](https://vuejs.org/style-guide/).
```
Comment (nzakas) [inline src/content/drafts/vue-rules-better-life.md:25]: Some cleanup, and the last sentence doesn’t fit with the paragraph or the rest of the article. 

```suggestion
I spent some time digging into the uncategorized category to see which rules might be useful to help avoid problems in commits and to enforce the same style in all parts of a Vue project. In my opinion, all Vue projects should use the Recommended category, which enforces a lot (if not all) of the topics mentioned in the [style guide](https://vuejs.org/style-guide/).
```

**Review by nzakas — CHANGES_REQUESTED** [review]
Thanks for submitting. This is pretty light on content, so as-is, we can’t accept it for the blog. While we don’t mind opinion pieces, it seems like there is nothing more than “this is how I like to configure these rules,” which I’m not sure is enough for a post.

I’d suggest thinking through a few questions to clarify if you think there is a story here:

- Why would ESLint users want to read this? What benefit are they receiving? Why would they read this instead of something else?
- Same for Vue users?
- What are the benefits you’ve seen, not just “it makes code the way I like it,” but practically speaking? Did you reduce bugs? Did it help in another way?
- What is the conclusion you’d like readers to take away?

**Comment by nzakas** [comment]
Closing as we won't be moving forward here.

**Pull request closed without merging**